### PR TITLE
Support stats of all  executor in graphd

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: src/graph/Executor.h [hunk 1]
```diff
@@ -28,8 +28,11 @@ namespace graph {
 
 class Executor : public cpp::NonCopyable, public cpp::NonMovable {
 public:
-    explicit Executor(ExecutionContext *ectx) {
+    explicit Executor(ExecutionContext *ectx, const std::string statsName) {
```
Comment (yixinglu) [inline src/graph/Executor.h]:
```cpp
const std::string &statsName = ""
```

File: src/graph/PipeExecutor.cpp [hunk 2]
```diff
@@ -11,7 +11,7 @@ namespace nebula {
 namespace graph {
 
 PipeExecutor::PipeExecutor(Sentence *sentence,
-                           ExecutionContext *ectx) : TraverseExecutor(ectx) {
+                           ExecutionContext *ectx) : TraverseExecutor(ectx, "") {
```
Comment (yixinglu) [inline src/graph/PipeExecutor.cpp]: Use default parameter: `TraverseExecutor(ectx)`

File: src/graph/SequentialExecutor.cpp [hunk 3]
```diff
@@ -14,7 +14,7 @@ namespace nebula {
 namespace graph {
 
 SequentialExecutor::SequentialExecutor(SequentialSentences *sentences,
-                                       ExecutionContext *ectx) : Executor(ectx) {
+                                       ExecutionContext *ectx) : Executor(ectx, "") {
```
Comment (yixinglu) [inline src/graph/SequentialExecutor.cpp]: ditto

Comment (yixinglu) [reply src/graph/SequentialExecutor.cpp]: the default empty string also can be removed here.

**Comment by Shylock-Hg** [comment]
Add lock require for #1444

**Pull request merged**